### PR TITLE
A reason for cargo to disarm anomaly cores.

### PR DESCRIPTION
**Pull request by davethwave**
This will basically just add price tags to anomaly cores, the more dangerous they are to disarm, the more money they are worth. Just only have one core price set up right now but I'll get the rest done later.

:cl: davethwave
add: You can now sell anomaly cores
/:cl:

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Way too much money considering how fast engineering can generate them.

Would prefer for a single core to be a bounty

**Comment by CPTANT** [comment]
Honestly I think anomaly cores shouldn't drop from supermatter delamination anomalies.

File: code/modules/cargo/exports/parts.dm [hunk 1]
```diff
@@ -19,3 +19,8 @@
 	cost = 2000
 	unit_name = "deactivated alien deconstruction drone"
 	export_types = list(/obj/item/deactivated_swarmer)
+
+/datum/export/pyroclastic_anomaly
+	cost = 10000
+	unit_name = "A rare disarmed anomaly core"
+	export_types = list(/obj/effect/anomaly/pyro)
```
Comment (RandomMarine) [inline code/modules/cargo/exports/parts.dm]: Is this some kind of experiment to see if anyone bothers to read the diff?

Comment (KorPhaeron) [reply code/modules/cargo/exports/parts.dm]: nice

**Comment by Tlaltecuhtli** [comment]
does only pyro sell?

**Comment by optimumtact** [comment]
what kor said, this can just be a bounty when pubby is finished.

**Comment by davethwave** [comment]
Understood

**Pull request closed without merging**